python/lang/security/insecure-uuid-version.yaml

```
@@ -0,0 +1,31 @@
+rules:
+  - id: insecure-uuid-version
+    patterns:
+      - pattern: uuid.uuid1(...)
+    message: |
+      Using UUID version 1 for UUID generation can lead to predictable UUIDs based on system information (e.g., MAC address, timestamp). This may lead to security risks such as the sandwich attack e.g. https://www.landh.tech/blog/20230811-sandwich-attack/. Consider using `uuid.uuid4()` instead for better randomness and security.
+    metadata:
+      cwe: 
```

```suggestion
    message: |
      Using UUID version 1 for UUID generation can lead to predictable UUIDs based on system information (e.g., MAC address, timestamp). This may lead to security risks such as the sandwich attack. Consider using `uuid.uuid4()` instead for better randomness and security.
    metadata:
      references:
        - https://www.landh.tech/blog/20230811-sandwich-attack/
      cwe: 
```

Hi @mohamed-osama-aboelkheir . Thanks for your contribution, this rule looks solid. 
Before merging, can I ask you to remove the url from the message and add it to the metadata?